client/js/components/button/BackToTopButton.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <dp-button
+    class="sticky z-above-zero"
```

Comment:
I think I found the reason, why the position is not a good Idea. We should move it to the right again
![Screenshot_2023-07-24_09-51-19-grundeinstellungen-btt](https://github.com/demos-europe/demosplan-core/assets/40487461/cd8492d3-5e06-416b-8342-08d31884c1a0)